frontend/src/app/types/states/authState.ts

```diff
@@ -0,0 +1,7 @@
+import { Token } from '../auth/token';
+import { User } from '../user/user';
+
+export interface AuthState {
```

Comment:
State is a part of store. Should be moved there

Reply:
Fixed